### PR TITLE
game view guides only if inspector expanded, freeLook rigs in tabs

### PR DESCRIPTION
**Pull request by glabute**
### Purpose of this PR

CMCL-287
Problem: with FramingTransposer/Composer combos and with FreeLooks, multiple game-view guides are drawn on top of each other, and it's difficult to read and to manipulate.

Idea: Only draw the guide if the component subsection is expanded.  That way the user can choose which component to expand and work on, and get the right guides.

For the FreeLook, put the rigs in tabs - only the selected rig has the guides visible.

### Testing status

- [ ] Added an automated test
- [ ] Passed all automated tests
- [x] Manually tested 

### Documentation status

- [x] Updated [CHANGELOG](https://keepachangelog.com/en/1.0.0/)
- [ ] Updated README (if applicable)
- [ ] Commented all public classes, properties, and methods
- [ ] Updated user documentation

### Technical risk

low

### Comments to reviewers

Look also at the CIinemachineNewVirtualCamera  and CinemachineNewFreeLook classes

### CODE REVIEW

**Comment by gaborkb** [comment]
The new UI is great! I am liking it! 👍

File: Editor/Utility/InspectorUtility.cs [hunk 1]
```diff
@@ -192,4 +192,27 @@ internal static string GetVirtualCameraObjectName(SerializedProperty property)
             return obj.name;
         }
     }
+
+    /// <summary>
+    /// Hack for vcams because there's no way to find out if an editor is collaped
```
Comment (gaborkb) [inline Editor/Utility/InspectorUtility.cs:197]: I think, it's a nice hack :)

File: Editor/Editors/CinemachineFreeLookEditor.cs [hunk 2]
```diff
@@ -95,27 +92,33 @@ public override void OnInspectorGUI()
...
-            if (m_rigs == null)
-                m_rigs = new CinemachineVirtualCameraBase[RigNames.Length];
-            if (m_editors == null)
-                m_editors = new UnityEditor.Editor[RigNames.Length];
-            for (int i = 0; i < RigNames.Length; ++i)
+            new GUIContent("Top Rig"), 
+            new GUIContent("Middle Rig"), 
+            new GUIContent("Bottom Rig")
+        };
+        int m_SelectedRig = 1;
```
Comment (gaborkb) [inline Editor/Editors/CinemachineFreeLookEditor.cs:101]: This should be static to be consistent with NewFreelook, and vcams FTransposer or Composer

File: Editor/Editors/CinemachineFreeLookEditor.cs [hunk 3]
```diff
@@ -95,27 +92,33 @@ public override void OnInspectorGUI()
...
+                if (rig != null)
                 {
-                    m_rigs[i] = rig;
-                    if (m_editors[i] != null)
-                        UnityEngine.Object.DestroyImmediate(m_editors[i]);
-                    m_editors[i] = null;
-                    if (rig != null)
-                        CreateCachedEditor(rig, null, ref m_editors[i]);
+                    Undo.RecordObject(Target, "selected rig");
+                    Target.m_YAxis.Value = m_SelectedRig == 0 ? 1 : (m_SelectedRig == 1 ? 0.5f : 0);
```
Comment (gaborkb) [inline Editor/Editors/CinemachineFreeLookEditor.cs:120]: This puzzled me for a while, but now I understand what it does. When you select a rig, this activates the selected rig by moving the vcam rig on the y axis.

**Review by gaborkb — CHANGES_REQUESTED** [review]
Small change for consistency.

File: Editor/Editors/CinemachineFreeLookEditor.cs [hunk 4]
```diff
@@ -76,16 +75,15 @@ public override void OnInspectorGUI()
             // Rigs
             if (Selection.objects.Length == 1)
             {
-                UpdateRigEditors();
-                for (int i = 0; i < m_editors.Length; ++i)
+                EditorGUILayout.Separator();
+                EditorGUILayout.Separator();
```
Comment (gaborkb) [inline Editor/Editors/CinemachineFreeLookEditor.cs:79]: Are the two separators intentional?

File: Editor/Editors/CinemachineVirtualCameraEditor.cs [hunk 5]
```diff
@@ -36,6 +31,15 @@ public bool IsExpanded
...
         bool IsPrefab { get; set; }
 
+        GUIContent m_ProceduralMotionLabel = new GUIContent(
+            "Procedural Motion", 
+            "Use the procedural motion algorithms to automatically drive the transform in "
+                + "relation to the LookAt and Follow targets.  \n\n"
+                + "Body controls the position, and Aim controls the rotation.\n\n"
+                + "If Do Nothing is selected, "
+                + "then the transform will not be written to, and can be controlled manually "
+                + "or otherwise driven by script.");
```
Comment (gaborkb) [inline Editor/Editors/CinemachineVirtualCameraEditor.cs:41]: This is very useful!

**Review by gaborkb — APPROVED** [review]

File: Editor/Editors/CinemachineVirtualCameraEditor.cs (hunk 5 above)
Comment (glabute) [reply Editor/Editors/CinemachineVirtualCameraEditor.cs:41]: yes

File: Editor/Editors/CinemachineFreeLookEditor.cs (hunk 4 above)
Comment (glabute) [reply Editor/Editors/CinemachineFreeLookEditor.cs:79]: yes

**Review by gaborkb — APPROVED** [review]

File: Editor/Editors/CinemachineComposerEditor.cs [hunk 6]
```diff
@@ -50,26 +54,28 @@ public override void OnInspectorGUI()
...
+            // Don't draw the guides if rendering to texture
             var vcam = Target.VirtualCamera;
             CinemachineBrain brain = CinemachineCore.Instance.FindPotentialTargetBrain(vcam);
             if (brain == null || (brain.OutputCamera.activeTexture != null && CinemachineCore.Instance.BrainCount > 1))
                 return;
 
             // Screen guides
-            bool isLive = brain.IsLive(vcam, true);
-            mScreenGuideEditor.OnGUI_DrawGuides(isLive, brain.OutputCamera, Target.VcamState.Lens, true);
+            bool isLive = targets.Length <= 1 && brain.IsLive(vcam, true);
```
[the remark this answers is not in the document]
Comment (glabute) [reply Editor/Editors/CinemachineComposerEditor.cs:77]: We don't draw draggable guides for multi-select

**Pull request merged**